site/app/models/Gradeable.php

```diff
@@ -485,6 +491,21 @@ protected function loadGradeableConfig() {
           $num_textboxes = count($details['textboxes']);
         }
 
+        if (isset($details['required_capabilities'])){
+            $this->required_capabilities = $details['required_capabilities'];
+        }
+        else{
+            $this->required_capabilities = "default";
+        }
```

Comment:
`$this->required_capabilities = $details['required_capabilities'] ?? 'default';`